.github/workflows/build-common.yml

```diff
@@ -8,7 +8,7 @@ on:
         required: false
         
 env:
-  EXPORTER_VERSION: 1.0.0-beta.25 # to be updated with the latest version
+  EXPORTER_VERSION: 1.0.0-beta.26 # to be updated with the latest version
```

Comment:
you need to add exporter jar to this PR + the pom to the root folder.

Reply:
![image](https://github.com/user-attachments/assets/76ccf006-0291-4be4-bd21-a928f2e34c0b)

like this.. you need 1.0.0-beta.26 instead.

Reply:
just added the jar and pom file to the root folder.